Keas.Mvc/Controllers/Api/DocumentController.cs

```diff
@@ -16,6 +16,8 @@ namespace Keas.Mvc.Controllers.Api
     [Authorize(Policy = AccessCodes.Codes.DocumentMasterAccess)]
     [ApiController]
     [Route("api/{teamName}/documents/[action]")]
+    [Consumes("application/json")]
```

Comment:
@srkirkland Does the Document controller need XML?

Reply:
nope, just the WebhookController, which isn't part of our swagger API surface so this change should be fine